server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -267,57 +271,115 @@ trait EmailQueryMethodContract {
          |  "methodCalls": [[
          |    "Email/query",
          |    {
-         |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6"
+         |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |      "comparator": [{
+         |        "property":"receivedAt",
+         |        "collation":"some thing not matter and ignored"
```

Comment:
i am not sure. It is a good idea to allow a fancy collation here.
The collation should be used only for String comparison.
And the value should be in a list returned by the server in the session capabilities see  #3744

Reply:
Let's only allow `i;unicode-casemap` as a value for now and reject all others

Reply:
`collation` removed